timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -93,6 +93,41 @@ types:
           inclusiveUpper: Long
       ConjureGetFreshTimestampsResponseV2:
         alias: ConjureTimestampRange
+      ConjureUnchangedChangeMetadata:
+        fields: {}
+      ConjureUpdatedChangeMetadata:
+        fields:
+          oldValue:
+            type: binary
+            safety: unsafe
+          newValue:
+            type: binary
+            safety: unsafe
+      ConjureDeletedChangeMetadata:
+        fields:
+          oldValue:
+            type: binary
+            safety: unsafe
+      ConjureCreatedChangeMetadata:
+        fields:
+          newValue:
+            type: binary
+            safety: unsafe
+      ConjureChangeMetadata:
+          union:
+            unchanged: ConjureUnchangedChangeMetadata
```

Comment:
Using knowledge from future PRs (time-travelling!), I know that Conjure's union types are not that efficient on the wire. However, I personally value staying consistent here more than moving to an even hackier wire format.